<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="minSlider">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="maxSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelMinSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Slider: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Max Slider: 0</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetTopRight" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>181</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDiff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock Difference</string>
   </property>
  </widget>
  <widget class="QSlider" name="ROISlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>391</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelROI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ROI: 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLeft">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUp">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>/\</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDown">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>\/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRight">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetTopLeft" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="lineButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
